silkrpc/concurrency/context_pool_test.cpp

```diff
@@ -57,7 +57,7 @@ TEST_CASE("Context", "[silkrpc][context_pool]") {
             Context context{create_channel, block_cache, state_cache, wait_mode};
             std::atomic_bool processed{false};
             auto* io_context = context.io_context();
-            io_context->post([&]() {
+            post(io_context->get_executor(), [&]() {
                 processed = true;
                 context.stop();
             });
```

Comment:
I would change this as:
```
boost::asio::post(*io_context, [&]() {
    processed = true;
    context.stop();
});
```
because 1) it makes clearer which `post` function we're calling 2) it is more compact using the overload taking `boost::asio::context&` as first parameter.

Reply:
ok